ocd-3220.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE openchpl.pending_surveillance_requirement DROP COLUMN IF EXISTS certification_criterion_id;
+ALTER TABLE openchpl.pending_surveillance_nonconformity DROP COLUMN IF EXISTS certification_criterion_id;
+ALTER TABLE openchpl.nonconformity_type_statistics DROP COLUMN IF EXISTS certification_criterion_id;
+
+ALTER TABLE openchpl.pending_surveillance_requirement
+ADD COLUMN certification_criterion_id bigint;
+
+ALTER TABLE openchpl.pending_surveillance_requirement
+ADD CONSTRAINT certification_criterion_fk FOREIGN KEY (certification_criterion_id)
+		REFERENCES openchpl.certification_criterion (certification_criterion_id)
+		MATCH FULL ON DELETE CASCADE ON UPDATE CASCADE;
+
+ALTER TABLE openchpl.pending_surveillance_nonconformity
+ADD COLUMN certification_criterion_id bigint;
+
+ALTER TABLE openchpl.pending_surveillance_nonconformity
+ADD CONSTRAINT certification_criterion_fk FOREIGN KEY (certification_criterion_id)
+		REFERENCES openchpl.certification_criterion (certification_criterion_id)
+		MATCH FULL ON DELETE CASCADE ON UPDATE CASCADE;
+
+ALTER TABLE openchpl.nonconformity_type_statistics
+ADD COLUMN certification_criterion_id bigint;
+
+ALTER TABLE openchpl.nonconformity_type_statistics
+ADD CONSTRAINT certification_criterion_fk FOREIGN KEY (certification_criterion_id)
+		REFERENCES openchpl.certification_criterion (certification_criterion_id)
+		MATCH FULL ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
Could you put a newline on this line? When the deployment script runs it joins all of the ocd-nnnn files together, and having newlines at the end of each make the vM.N.O script look a little nicer

It's not required, because the script does assume people _might_ not have those newlines

I'll approve as written